docs/reference/monitoring/troubleshooting.asciidoc

```diff
@@ -27,3 +30,14 @@ monitoring data by using {metricbeat} the indices have `-mb` in their names. If
 the indices do not exist, review your configuration. For example, see
 <<monitoring-production>>.
 
+[[monitoring-troubleshooting-uuid]]
+=== Some monitoring data is missing from {kib}
+
+*Symptoms*:
+The information about some nodes, instances, or Beats in your cluster are
```

Comment:
If you're talking about ES nodes, LS nodes, etc, here, then you should spell out the list like you do below.

For American English, the verb here should be singular (information...is missing).

Reply:
I decided not to repeat all the terms, but I've reformatted the sentence and addressed the typo, thanks!